### PR TITLE
Split emscripten circle job into build and test.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Comment by chriseth** [comment]
This way we see whether the build or the test fails. We might actually even split out the zeppelin tests into their own circle job.

**Comment by axic** [comment]
@chriseth does this commit "Make all lookups recursive by default." belong to this PR?

File: test/solcjsTests.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    npm config set script.prepublish ''
+    npm install
+
+    # Replace soljson with current build
+    echo "Replacing soljson.js"
+    rm -f soljson.js
+    cp "$SOLJSON" soljson.js
+
+    # Update version (needed for some tests)
+    VERSION=$("$REPO_ROOT/scripts/get_version.sh")
```
Comment (axic) [inline test/solcjsTests.sh]: This script is missing the definition of `REPO_ROOT`.

**Comment by axic** [comment]
Can you also double check how will the #2818 PR work after the split of the scripts?

**Comment by chriseth** [comment]
#2818 should be fine. I can rebase that once this is merged.

File: test/solcjsTests.sh (hunk 1 above)
Comment (chriseth) [reply test/solcjsTests.sh]: yes, should be fixed now

**Comment by chriseth** [comment]
Ah sorry, will remove the "recursive" commit.

**Comment by axic** [comment]
Double checked myself, I thought it changed the `solcTests` part, but it should be independent. Good then :)

**Review by axic — APPROVED** [review]

**Pull request merged**